src/main/java/com/GG/T9AgeCombat/predicates/DetermineModificationPredicate.java

```diff
@@ -9,10 +9,26 @@
     private DetermineModificationPredicate() {
     }
 
-    public static void applyBonus(Unit unit, Modification modification, Integer value) {
+    public static void applyBonus(Unit unit, Modification modification, Object value) {
         switch (modification) {
-            case STRENGTH:
-                unit.updateStrength(value);
+            case STRENGTH: {
+                unit.updateStrength((Integer) value);
```

Comment:
We had talked about adding a strengthModifier field to the Unit because we may want to apply the rule in some cases but not necessarily change the underlying value.

I think this is good for now until we really dive into that.

Reply:
Yes I was thinking we would do that next step when we implement using the modified strength